hooks/blueprint

```diff
@@ -1,25 +1,44 @@
 #!/bin/bash
-set -eu
+#set -eu
 
 declare -a merge
 merge=( "manifests/concourse/base.yml" )
+merge+=( "manifests/concourse/releases.yml")
+merge+=( "manifests/concourse/jobs.yml")
 
 validate_features \
   azure shield                                  \
-  workers full                                  \
+  workers full small-footprint                  \
   no-tls provided-cert self-signed-cert         \
   github-oauth github-enterprise-oauth cf-oauth \
   vault shout
 
 if [[ "$(bosh_cpi)" == "azure" ]]; then
   merge+=( "manifests/addons/azure.yml" )
 fi
-if want_feature "full" ; then
-  if want_feature "workers" ; then
-    echo >&2 "Can only have one of 'full' or 'workers' as a feature."
+
+# Make sure we're only merging one of the major feature flags
+declare -i maj_feat=0
+for __feature in $GENESIS_REQUESTED_FEATURES; do
+  if [[ $__feature == "full" || $__feature == "workers" || $__feature == "small-footprint" ]]; then
+    (( maj_feat++ ))
+  fi
```

Comment:
It's probably easier to do a series of `want_feature X` than to loop over `$GENESIS_REQUESTED_FEATURES`